src/API.jl

```diff
@@ -455,7 +455,15 @@ function test(ctx::Context, pkgs::Vector{PackageSpec};
     return
 end
 
-is_manifest_current(ctx::Context = Context()) = Operations.is_manifest_current(ctx.env)
+is_manifest_current(ctx::Context) = Operations.is_manifest_current(ctx.env)
```

Comment:
It broke support of Julia 1.11 in Quarto (which is fixed by now) 😄 Generally, shouldn't documented functionality in Pkg.API be stable and not be broken in semver-nonbreaking releases? Maybe this should be changed to
```suggestion
is_manifest_current(ctx::Context = Context()) = Operations.is_manifest_current(ctx.env)
```
again?

Reply:
I don't have a strong opinion about this but if you managed to find a working use case for it I guess it has some value. But I'd suggest opening a new PR rather than adding a change suggestion to a merged one.